src/ngx-smart-modal/src/components/ngx-smart-modal.component.ts

```diff
@@ -8,8 +8,9 @@ import {
   EventEmitter,
   HostListener,
   ChangeDetectorRef,
-  ViewChild,
-  ElementRef
+  ViewChildren,
+  ElementRef,
+  QueryList
```

Comment:
So if you keep ViewChild, you don't need `ViewChildren` and `QueryList`

Reply:
Nice catch! 💪